src/Abstractions/NexusMods.Abstractions.Loadouts.Synchronizers/ALoadoutSynchronizer.cs

```diff
@@ -1278,8 +1278,8 @@ private async ValueTask<Hash> MaybeHashFile(IDb hashDb, GamePath gamePath, Absol
             if (hash.MinimalHash == diskMinimalHash)
                 return hash.XxHash3;
         }
-        
-        // If we didn't find a match, then we need to hash the file
+
+        _logger.LogDebug("Didn't find matching hash data for file `{Path}`, falling back to doing a full hash", file);
```

Comment:
This sounds like it would trigger a lot, for any non game file basically?

Reply:
Every file we don't have hashes for, that's why I made this a debug log.

Reply:
I wouldn't be surprised to see thousands of these, but sure